### DISCUSSION TITLE
how to use ssr in v6.4

### DISCUSSION THREAD

**Discussion by KyrieLii** [remix-run/react-router#9344] [Q&A]
> https://reactrouter.com/en/main/guides/ssr  

the docs is need update , in csr i use `createBrowserRoute` like demos: 

```typescript
const router = createBrowserRouter([
  {
    path: "/",
    element: <Root />,
    loader: rootLoader,
    children: [
      {
        path: "team",
        element: <Team />,
        loader: teamLoader,
      },
    ],
  },
]);

ReactDOM.createRoot(document.getElementById("root")).render(
  <RouterProvider router={router} />
);
```

but how to use `StaticRouter`   ?

```typescript
<StaticRouter>
  <RouterProvider router={router} />
</StaticRouter>
```

**Comment by GiveMe-A-Name** [comment]
you can watch those examples.
- https://github.com/remix-run/react-router/tree/main/examples/ssr-data-router
- https://github.com/remix-run/react-router/tree/main/examples/ssr
